test-fixtures/data/my-acl-1.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I really like the fact that the file formats are close to the format used to send updates to the API.  How do you feel about adding the [`"op"` field](https://github.com/fastly/api-planning/blob/main/proposals/new-apis/acls_in_compute.md#input-json-parameters-1) too, so that the formats are the same?

Reply:
Yes, good idea. I've made that change in [c18e3b6](https://github.com/fastly/Viceroy/pull/438/commits/c18e3b6e8234c0117b1e64ffc3078ab7079706e8)